tests/analytics/test_analytics.py

```diff
@@ -0,0 +1,70 @@
+from uuid import UUID
+
+import pytest
+
+from great_expectations.analytics.config import (
+    DUMMY_UUID,
+    Config,
+    get_config,
+    update_config,
+)
+from great_expectations.analytics.events import DataContextInitializedEvent
+
+TESTING_UUID = UUID("00000000-c000-0000-0000-000000000000")
+
+
+@pytest.fixture(
+    scope="function",
+    params=[
+        (
+            Config(
+                organization_id=TESTING_UUID,
+                user_id=TESTING_UUID,
+                data_context_id=DUMMY_UUID,
+                oss_id=DUMMY_UUID,
+                cloud_mode=False,
+            ),
+            TESTING_UUID,
+            {
+                "user_id": TESTING_UUID,
+                "organization_id": TESTING_UUID,
+                "data_context_id": DUMMY_UUID,
+                "oss_id": DUMMY_UUID,
+                "service": "gx-core",
+            },
+        ),
+        (
+            Config(),
+            DUMMY_UUID,
+            {"data_context_id": DUMMY_UUID, "oss_id": DUMMY_UUID, "service": "gx-core"},
+        ),
+    ],
+)
+def analytics_config(request):
+    base_config = get_config()
+    update_config(request.param[0])
+    yield request.param[1], request.param[2]
+    update_config(base_config)
+
+
+def test_event_identifiers(analytics_config):
```

Comment:
Could we annotate with `@pytest.mark.unit`?

Reply:
I need a primer on the pytest markings -- I don't think there were any other tests for analytics, and I'm not sure. So, I'll add that annotation, but please to tell me why!

Reply:
Oh, nice, we even had an automated check for this!

Reply:
I think the `pyproject.toml` lists all the available ones but maybe we need an analytics one if we start going E2E